Todo.Web/Server/Todo.Web.Server.csproj

```diff
@@ -4,10 +4,14 @@
     <TargetFramework>net7.0</TargetFramework>
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
+    <ContainerImageName>todo-web-server</ContainerImageName>
+    <ContainerImageTag>latest</ContainerImageTag>
+    <UserSecretsId>7019fce6-4c5b-45c3-9f24-5635eae3f449</UserSecretsId>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.Server" Version="7.0.0" />
+    <PackageReference Include="Microsoft.NET.Build.Containers" Version="0.2.7" />
```

Comment:
This package is currently required to publish an app as a container according with this [doc](https://learn.microsoft.com/en-us/dotnet/core/docker/publish-as-container).